src/components/Sidebar/Sidebar.tsx

```diff
@@ -1,29 +1,120 @@
 import React from 'react';
+import { useState } from 'react'
 import './Sidebar.scss';
 import { GlobalProps } from '../app/App';
+import logo from '../../images/logo.svg'
+import logotext from '../../images/logo-text.svg'
+import feed_normal from '../../images/feed_normal.svg'
+import feed_selected from '../../images/feed_selected.svg'
+import explore_normal from '../../images/search_normal.svg'
+import explore_selected from '../../images/search_selected.svg'
+import notif_normal from '../../images/notification_normal.svg'
+import notif_selected from '../../images/notification_selected.svg'
+import direct_normal from '../../images/direct_normal.svg'
+import direct_selected from '../../images/direct_selected.svg'
```

Comment:
You again did not see, the changes that I made.
1. Do not import images, it serves no purpose, instead simply pass the URL where it's needed.
2. Do  not pass the relative URL, see what have I passed on avatar component.

Please click on the commits I make, and see what changes I did.
It's time taking, but worth it.

Reply:
Okay, It didn't feel right to pass hard coded string path. I thought at least the IDE would inform if it doesn't find the asset in the place.

I understand point 2, are you sure that point 1 is best practice?

Reply:
Hey @ishan-srivastava, I think it is better in the current form.

For eg. I tried moving an image to another directory, IDE automatically refactored the import to the new location.
Whereas, if I hardcode it as `url("/assets/images/placeholder-dp.png")`, it doesn't get refactored. Neither `CMD` + `Click` worked, it says "Cannot find declaration to go".

Now, I mistakenly put all images into `src/images/` and I want to move all to `assets/images/`, since I used import, IDE will refactor code for all 16 images automatically, else I would have to go into each hardcoded argument and change path.

What do you think about this argument?

Reply:
Sounds good, maybe we can make a barrel file in the images folder, that can solve both our problems.
Please add one, here is a reference for you to read.
https://dev.to/luispa/how-to-create-barrels-in-typescript-or-javascript-59ma